charts/envoy/ci/test-values.yaml

```diff
@@ -0,0 +1,98 @@
+# replicaCount -- number of replicas to deploy
```

Comment:
This will be the 'values.yaml' file for testing.
It is used by Kubeval and chart-testing.